src/atomizer.js

```diff
@@ -678,11 +684,6 @@ Atomizer.prototype.getCss = function (config/*:AtomizerConfig*/, options/*:CSSOp
         }
     });
 
-    if (options.namespace) {
-        var jssNew = {};
-        jssNew[options.namespace] = jss;
-        jss = jssNew;
-    }
     if (options.helpersNS) {
```

Comment:
On a related note, I really think we need to rename the `helpersNS` option.  It's not consistently named compared to `namespace` for Atomic classes.  I'd rather we call it something like `namespace_helpers` (so it's visually grouped with `namespace`) or I suppose `helpers_namespace`.